src/eval-frame/actions/language-actions.js

```diff
@@ -136,7 +152,14 @@ export function runCodeWithLanguage(language, code, messageCallback) {
   if (asyncEvaluator !== undefined) {
     const messageCb =
       messageCallback === undefined ? () => {} : messageCallback;
-    return window[module][asyncEvaluator](code, messageCb);
+    try {
+      return window[module][asyncEvaluator](code, messageCb);
+    } catch (e) {
+      if (e.message === "window[module] is undefined") {
+        return new Error(`eval type ${module} not not defined`);
```

Comment:
i think we can pull `displayName` out of `language` on line 151 in order to make this message clearer for end-users, maybe somthing like:

`Error evaluating ${language.displayName}; evaluation module "${module}" not not defined`

Reply:
sure